site/gatsby-site/cypress/integration/submit.js

```diff
@@ -586,4 +587,14 @@ describe('The Submit form', () => {
 
     cy.contains('.invalid-feedback', '*Incident Date required').should('be.visible');
   });
+
+  maybeIt('Should show the editor notes field when logged in', () => {
+    cy.login(Cypress.env('e2eUsername'), Cypress.env('e2ePassword'));
+    cy.visit(url);
+    cy.get('[name="editor_notes"').should('exist');
+  });
+  maybeIt('Should not show the editor notes field when not logged in', () => {
```

Comment:
This `maybeIt` function (which is a horrible name, sorry) should only be used when the test requires the user to be logged in.

Reply:
I believe it does require the user to be logged in? The editor_notes field is only supposed to show up for logged in users, since it's sort of a project management feature more than one for the general public. So this test and the next are checking that it only shows up for logged-in users.

Reply:
As it is this test will be skipped on production (something that we'll fix in the future).

`maybeIt` should be used only with tests that use the login command inside.

(I'm talking about `Should not show the editor notes field when not logged in`)

Reply:
Ah, got it. Updated. Thanks!